### ISSUE TITLE
Scrollbar size in firefox

### ISSUE THREAD

**Issue by exzemat** [dirruk1/gnome-breeze#55]
Hello,
Scrollbar in fiirefox are thinner than scrollbar in other place.
Look screenshot firefox/dolphin just to compare.
But it's flagrant in full screen
![screenshot_20160316_212137](https://cloud.githubusercontent.com/assets/2069462/13827581/a310aae8-ebbd-11e5-8838-d5a637a107da.png)
In any case, thanks

**Comment by ghost** [comment]
Hm, it looks like the Firefox's GTK 3 implementation, which is somehow no longer considered experimental despite having some serious bugs. The arrows on the scrollbars you see there shouldn't appear either.

I'll have to do some digging to find some reliable workarounds for this, and it will probably be postponed to the GTK 3.20+ versions of the theme. Thanks for bringing it to our attention.

**Comment by shmerl** [comment]
Yeah, I have the same bug in Firefox 46 (beta 11) which already enabled GTK3, and it which will be released with GTK3 support. Another issue there is very huge spacing between menu items which I can only reduce with userChrome.css in Firefox. I'll open a separate bug about it.

**Comment by Mte90** [comment]
In my case using firefox 47 beta also with the 48 alpha, I have the arrow but the scrollbar it is not showed.
Also in firefox the checkbox and radio are not rendered, only if they are checked.

**Comment by shmerl** [comment]
Not shown scrollbar and radio buttons is a problem of compatibility wtih gtk 3.20. It's tracked in bug #56.
